addons/account/views/product_view.xml

```diff
@@ -7,25 +7,28 @@
             <field name="priority">5</field>
             <field name="inherit_id" ref="product.product_template_form_view"/>
             <field name="arch" type="xml">
-                <page name="sales" position="after">
-                    <page string="Invoicing" name="invoicing" groups="account.group_account_invoice">
+                <page name="inventory" position="after">
+                     <page string="Accounting" name="accounting" groups="account.group_account_user">
                         <group name="properties">
-                            <group>
+                            <group string="Receivables">
                                 <field name="property_account_income_id"
                                     domain="[('internal_type','=','other'),('deprecated','=',False)]"
                                     groups="account.group_account_user"/>
-                                <field name="taxes_id" widget="many2many_tags"/>
                             </group>
-                            <group>
+                            <group string="Payables">
                                 <field name="property_account_expense_id"
                                     domain="[('internal_type','=','other'),('deprecated','=',False)]"
                                     groups="account.group_account_user"/>
-                                <field name="supplier_taxes_id" widget="many2many_tags"/>
                             </group>
                         </group>
                         <group name="accounting"/>
                     </page>
                 </page>
+                <xpath expr="//group[@name='email_template_and_project']" position="before">
```

Comment:
You can directly write <group name="email_template_and_project" position="before"> .